### PR TITLE
chore: improve readability of TOC in functions pages

### PR DESCRIPTION
**Pull request by bsmth**
__Description:__
* Removed `Description` sections in favor of including this in the top-level description of a function to reduce redundant information
* Replaced section headings with bold text for:
  * Examples
  * Arguments
  * Return Values

### CODE REVIEW

File: docs/reference/function/random-value-generator.md [hunk 1]
```diff
@@ -201,11 +180,11 @@ null,null,null,null,null
...
-- `rnd_long()` - generates a random `long` value.
-- `rnd_long(min, max, nanRate)` - generates a random `long` value within a range
-  which can be `NaN`.
-
-### Arguments
+- `rnd_long()` is used to return a random signed integer between
+  `0x8000000000000000L` and `0x7fffffffffffffffL`.
+- `rnd_long(min, max, nanRate)` is used to generate long values in a specific
+  range (for example only positive, or between 1 and 10), or to get occasional
+  `NaN` values along with int values. **Arguments:**
```
Comment (mpsq) [inline docs/reference/function/random-value-generator.md:187]:
```suggestion
  `NaN` values along with int values.
  
  **Arguments:**
```

File: docs/reference/function/row-generator.md [hunk 2]
```diff
@@ -47,7 +45,7 @@ all random functions.
 
 :::
 
-### Examples
+## Examples
```
Comment (mpsq) [inline docs/reference/function/row-generator.md:48]:
```suggestion
**Examples:**
```

**Review by mpsq — CHANGES_REQUESTED** [review]
minor changes, it looks good otherwise

File: docs/reference/function/random-value-generator.md (hunk 1 above)
Comment (bsmth) [reply docs/reference/function/random-value-generator.md:187]: thanks, good catch

**Comment by bsmth** [comment]
Closes #345

**Review by mpsq — APPROVED** [review]
lgtm

**Pull request merged**